_content/monographs/kamma-in-buddhism_buddhadasa.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Kamma in Buddhism"
+authors:
+  - buddhadasa
+translator: Santikaro
+external_url: "https://www.suanmokkh.org/system/books/files/000/000/089/original/Buddhadas-Kamma-in-buddhism.pdf?1599464247"
+drive_links:
+  - "https://drive.google.com/file/d/1OdOqbmOuzO48q0a7tR508jvGMIgo2REX/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
+address: "Wisconsin"
+volume: 
+series: Messages from Suan Mokkh Series
+number: 4
+pages: 22
```

Comment:
only 8 pages of actual content... And even those are half size (A5) so I'd honestly say `pages: 4`